packages/lightning-lsp-common/src/__tests__/workspace-type.test.ts

```diff
@@ -0,0 +1,209 @@
+import mockFs from 'mock-fs';
+import { detectWorkspaceType, WorkspaceType } from '../shared';
+
+describe('detectWorkspaceType', () => {
+    beforeAll(() => {
+        console.log = jest.fn();
```

Comment:
Why do you mock `console.log` and console.error`?

Reply:
Its a mockFS issue. If you do a console.log while mocking things, you get an exception.  I'll make a note.